### DISCUSSION TITLE
Allow to see cards from multiple projects

### DISCUSSION THREAD

**Discussion by ceolinwill** [community/community#19780] [Projects and Issues]
As an organization manager/maintainer, I'd love to be able to see/filter/group cards from multiple projects.

For example: I'd like to be able to go to the projects page, select multiple repositories (or all of them) and create a custom view for them where I can see, organization-wide, what the are the cards in progress, todo, etc.

In theory, this is going to be possible once https://github.com/github/roadmap/issues/274 is implemented since it would allow us to create a parent project for the organization and add all other projects as children of the main one.

However, I think it would be useful to have grouping projects as a standalone functionality, even when projects aren't linked to each other.

**Comment by huabin** [comment]
@wceolin 

Please look at this [issue](https://github.com/tourcoder/fsfg/issues/14), I think we have the same needs.
